### DISCUSSION TITLE
Sociedade Limitada Unipessoal + Remessa Online

### DISCUSSION THREAD

**Discussion by pocesar** [DyegoCosta/trabalhando-remoto#188] [General]
Alguém tem experiência com esse combo? Eu estou tentando entender, mas até meu contador tá meio perdido. A idéia é receber de empresa européia usando uma conta PJ de Sociedade Unipessoal Limitada, mas a Remessa Online ajusta o seu limite anual de acordo com seu IRPF, que desse ano a declaração vai ser bem menor do que ando recebendo atualmente (que foi a partir desse ano)...

**Comment by nobre84** [comment]
Desde maio de 2019 é possível criar LTDA sem sócios,b creio que seja o caso aí? Quanto à Remessa Online, eles teriam que aceitar o que estiver no seu contrato, não? Tá parecendo o Inter impondo limites arbitrários. Se for o caso troca o serviço, o BS2 tem sido bem avaliado por muitos. Tem também a transferwise, payoneer, entre outros.

**Comment by enyamada** [comment]
Até onde eu sei, o único limite que Remessa Online impõe é para o ENVIO de dinheiro para o exterior, e não o contrário.

**Comment by yuriploc** [comment]
Sou ME e tenho usado o Remessa Online sem problemas. Foi um alívio sair do Banco Inter! Meu "Independent Contractor" com a empresa gringa é com meu CPNJ e só precisei enviar uma vez. Os outros meses reuso o arquivo e envio a invoice e eles transferem para minha conta PJ, que ainda é no Inter.

**Comment by pocesar** [comment]
acabei de confirmar a conta PJ com eles, devo testar próximo mês ou em Agosto. o processo de abertura da conta foi bem rápido e todo à distancia (o que demorou mais foi o a certidão simplificada da junta comercial). minha conta PJ é da BS2, enquanto eles não fornecem conta internacional PJ também, pretendo usar o remessa online, o spread é muito bom (1,3%). no meu caso, eu sou LTDA como unipessoal

**Comment by g13ydson** [comment]
A remessa online só cobra a taxa de 1.3% + 0.83 de IOF mesmo?

**Comment by pocesar** [comment]
isso aí @g13ydson é muito abaixo do que banco que trata de cambio, cobra (0.38% de iof)

**Comment by marcelolima** [comment]
@pocesar vi que o Contabilizei tá com parceria com a BS2: https://www.contabilizei.com.br/conta-digital-pj/, pelo visto estão suportando recebimento do exterior.

**Comment by linck5** [comment]
@marcelolima Eu também uso a Contabilizei e tava pensando em fazer com a BS2. Mas parece que pra receber dinheiro você paga por transferência, USD 20, e mais 2% de spread. Eu por exemplo recebo em NZD, então se eu fosse receber NZD 5000 por exemplo, eu ia pagar esses USD 20 e mais de 300 reais nesse spread. Isso não é um pouco caro? Ou ta dentro do normal? Eu por exemplo quando usava TransferWise antes de abrir empresa, só pagava uma taxa, mas não tinha esse spread, era basicamente cotação com preço de mercado.

Pelo menos parece que no futuro a Contabilizei parece que vai ter umas integrações com a B2S que podem automatizar um pouco as coisas e deixar a nossa vida mais conveniente.

**Comment by lucas-barros** [comment]
@linck5 eu uso o serviço de cambio da BS2 e me parece ter as malhores taxas. Na verdade eles so cobrão o spread que varia entre 0.9% - 1%. Essa taxa de USD 20 pode ser cobrada por algum banco intermediário. Por exemplo, quando recebo transferência dos Estados Unidos não me é cobrado essa taxa, mas quando recebia da Europa, sim.

**Comment by praguc** [comment]
@pocesar e @yuriploc e qualquer pessoa que tiver a informação - vocês tiveram algum limite em recebimento com PJ.. eu não tenho certeza se o limite e somente para envio, ou para recebimento também? Qualquer dica, agradeço.

**Comment by pocesar** [comment]
@praguc eles limitam só o meu envio, que é metade do valor que eu recebi durante o ano todo, baseado na declaração de IRPF

**Comment by yuriploc** [comment]
@praguc Eles limitam o recebimento, mas caso você envie declaração de IR(PF/PJ) eles aumentam de acordo.
